lib/classes/PluginManager.js

```diff
@@ -352,6 +353,13 @@ class PluginManager {
     });
   }
 
+  assignDefaultOptions(command) {
+    _.forEach(command.options, (value, key) => {
+      if (value.default && (!this.cliOptions[key] || this.cliOptions[key] === true)) {
```

Comment:
😬 Thanks @dougmoscrop --> ⛹️ 

So as the author of this weird piece of code I'd say that we can drop 😂